src/platform/plugins/shared/dashboard/public/dashboard_actions/filters_notification_popover.tsx

```diff
@@ -77,9 +78,9 @@ export function FiltersNotificationPopover({ api }: { api: FiltersNotificationAc
     }
   }, [api, setDisableEditButton]);
 
-  const [dataViews, parentViewMode] = useBatchedOptionalPublishingSubjects(
-    api.parentApi?.dataViews$,
-    getViewModeSubject(api ?? undefined)
+  const [dataViews, parentViewMode] = useBatchedPublishingSubjects(
```

Comment:
Is there any good way to _ensure_ that parent is defined before we even get this far? Maybe by throwing an error earlier if the parent API isn't defined?

Reply:
Parent is not required. `parentViewMode` is only used to `showEditButton`.